src/drivers/bitbucket.go

```diff
@@ -9,8 +9,8 @@ import (
 	"github.com/git-town/git-town/v7/src/drivers/helpers"
 )
 
-// bitbucketCodeHostingDriver provides access to the API of Bitbucket installations.
-type bitbucketCodeHostingDriver struct {
+// BitbucketCodeHostingDriver provides access to the API of Bitbucket installations.
+type BitbucketCodeHostingDriver struct {
```

Comment:
I am avoiding a few unnecessary interface returns, which isn't good Go style. This requires exporting this struct.